src/components/TheBottomNav.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-bottom-navigation
```

Comment:
This should probably be wrapped in a footer tag for semantics.

Reply:
I wonder if Vuetify injects one?